MonoGame.Framework.Content.Pipeline/Graphics/FontDescriptionStyle.cs

```diff
@@ -14,6 +14,11 @@ namespace Microsoft.Xna.Framework.Content.Pipeline.Graphics
     public enum FontDescriptionStyle
     {
         /// <summary>
+        /// Normal text.
+        /// </summary>
+        Regular=0,
```

Comment:
You should write a unit test verifying this enum is correct to XNA.  Might want to setup a new test specifically for enums in the pipeline sort of like was done here:

https://github.com/mono/MonoGame/blob/develop/Test/Framework/EnumConformingTest.cs